packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -229,12 +251,15 @@ FlutterErrorDetails _debugReportException(
   DiagnosticsNode context,
   dynamic exception,
   StackTrace stack,
-) {
+  {
```

Comment:
I believe the correct formatting is to put the `{` on the previous line and keep the indentation on the next line the same.

Reply:
fixed